src/lib/market-service/market-service-manager.ts

```diff
@@ -142,4 +143,11 @@ export class MarketServiceManager {
     if (!result) return []
     return result
   }
+
+  async getPopularAssets(count: number): Promise<AssetId[]> {
```

Comment:
Can we give this a name that's closer to the actual network at hand? This is only getting popular assets because of *consumption*, but this is really finding all sorted by volume descending, the only reason it's getting popular assets is because the introduced and currently only consumption is using a certain count with the intent to get x most popular assets, but this could change.

Perhaps `findAllSortedByVolumeDesc` ?